sql-select/expensive-films.sql

```diff
@@ -0,0 +1,4 @@
+select *
+  from "films"
+order by "replacementCost"
+desc limit 10;
```

Comment:
For clarity, `desc` is associated with `order by` instead of `limit`, so it should normally go on that line.